lib/modules/showImages.js

```diff
@@ -904,8 +910,10 @@ function generateGallery(options) {
 
 	const preloadCount = parseInt(module.options.galleryPreloadCount.value, 10) || 0;
 
-	const filmstripActive = module.options.loadAllInAlbum.value;
-	const filmstripMaxCount = parseInt(module.options.dontLoadAlbumsBiggerThan.value, 10) || Infinity;
+	let filmstripActive = module.options.galleryAsFilmstrip.value;
+	const filmstripLoadIncrement = parseInt(module.options.filmstripLoadIncrement.value, 10) || Infinity;
+	const slideshowWhenLargerThan = parseInt(module.options.useSlideshowWhenLargerThan.value, 10) || Infinity;
+	if (filmstripActive) filmstripActive = options.src.length < slideshowWhenLargerThan;
```

Comment:
Move it down here and use `const`?

 `const filmstripActive = module.options.galleryAsFilmstrip.value && (options.src.length < slideshowWhenLargerThan);`